app/src/main/java/com/grzegorzdec/tagmango/seller/map/MapFragment.kt

```diff
@@ -0,0 +1,95 @@
+package com.grzegorzdec.tagmango.seller.map
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.google.android.gms.maps.*
+import com.grzegorzdec.tagmango.R
+import androidx.lifecycle.ViewModelProviders
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.grzegorzdec.tagmango.BaseFragment
+import com.grzegorzdec.tagmango.common.recyclerview.HorizontalMarginItemDecoration
+import com.grzegorzdec.tagmango.databinding.FragmentMapBinding
+
+
+const val MAPVIEW_BUNDLE_KEY = "MAPVIEW_BUNDLE_KEY"
+
+class MapFragment : BaseFragment() {
+
+    private lateinit var mapView: MapView
+    private var mapViewBundle: Bundle? = null
+
+    private val viewModel by lazy {
+        ViewModelProviders.of(this, MapFragmentViewModelFactory(repository))
+            .get(MapViewModel::class.java)
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val binding = FragmentMapBinding.inflate(inflater, container, false).apply {
+            viewModel = this@MapFragment.viewModel
+            recyclerView.addItemDecoration(
+                HorizontalMarginItemDecoration(
+                    resources.getDimensionPixelSize(R.dimen.client_item_offset)
+                )
+            )
+            recyclerView.layoutManager = LinearLayoutManager(context, LinearLayoutManager.HORIZONTAL, false)
+            recyclerView.itemAnimator = null
+            recyclerView.adapter = ClientsRecyclerAdapter(ViewModelProviders.of(this@MapFragment),  viewModel!!) {
+                this@MapFragment.viewModel.selectedClient = it
+            }
+        }
+        mapView = binding.map.apply {
+            onCreate(mapViewBundle)
```

Comment:
do we need this? if yes, move it to 
`FragmentMapBinding.inflate(inflater, container, false).apply {` block